apps/dashboard/app/apps/featured_app.rb

```diff
@@ -0,0 +1,12 @@
+# A simple subclass of OodApp that overrides category
+# and subcategory to allow for groupings other than what's
+# provided in the app's manifest & definition.
+class FeaturedApp < OodApp
+  attr_reader :category, :subcategory
+
+  def initialize(router, category: "Apps", subcategory: "Pinned Apps")
+    super(router)
+    @category = category.to_s
+    @subcategory = subcategory.to_s
+  end
```

Comment:
Okay for now, but we might consider a decorator instead in the future. For example:

```ruby
class CustomizableCategories < SimpleDelegator
  attr_writer :category, :subcategory
end

ood_app = CustomizableCategories.new(ood_app)
ood_app.category = "Apps"
ood_app.subcategory = "Pinned Apps"
```